etl/steps/data/garden/un/2025-03-12/migrant_stock.countries.json

```diff
@@ -0,0 +1,381 @@
+  "United Kingdom*": "United Kingdom",
+  "United Republic of Tanzania*": "Tanzania",
+  "United States Virgin Islands*": "United States Virgin Islands",
+  "United States of America*": "United States",
+  "Wallis and Futuna Islands*": "Wallis and Futuna",
+  "AFRICA": "Africa",
+  "ASIA": "Asia",
+  "American Samoa": "American Samoa",
+  "Anguilla": "Anguilla",
+  "Aruba": "Aruba",
+  "Australia": "Australia",
+  "Azerbaijan": "Azerbaijan",
+  "Bermuda": "Bermuda",
+  "Bolivia": "Bolivia",
+  "Bonaire Sint Eustatius and Saba": "Bonaire Sint Eustatius and Saba",
+  "British Virgin Islands": "British Virgin Islands",
+  "Brunei": "Brunei",
+  "Cape Verde": "Cape Verde",
+  "Cayman Islands": "Cayman Islands",
+  "Channel Islands": "Channel Islands",
+  "China": "China",
+  "Cook Islands": "Cook Islands",
+  "Cote d'Ivoire": "Cote d'Ivoire",
+  "Curacao": "Curacao",
+  "Cyprus": "Cyprus",
+  "Democratic Republic of Congo": "Democratic Republic of Congo",
+  "Denmark": "Denmark",
+  "EUROPE": "Europe",
+  "East Timor": "East Timor",
+  "Falkland Islands": "Falkland Islands",
+  "Faroe Islands": "Faroe Islands",
+  "Finland": "Finland",
+  "France": "France",
+  "French Guiana": "French Guiana",
+  "French Polynesia": "French Polynesia",
+  "Georgia": "Georgia",
+  "Gibraltar": "Gibraltar",
+  "Greenland": "Greenland",
+  "Guadeloupe": "Guadeloupe",
+  "Guam": "Guam",
+  "Hong Kong": "Hong Kong",
+  "Iran": "Iran",
+  "Isle of Man": "Isle of Man",
+  "Laos": "Laos",
+  "Macao": "Macao",
+  "Malaysia": "Malaysia",
+  "Martinique": "Martinique",
+  "Mauritius": "Mauritius",
+  "Mayotte": "Mayotte",
+  "Melanesia": "Melanesia",
+  "Micronesia": "Micronesia",
+  "Micronesia (country)": "Micronesia (country)",
+  "Moldova": "Moldova",
+  "Montserrat": "Montserrat",
+  "Netherlands": "Netherlands",
+  "New Caledonia": "New Caledonia",
+  "New Zealand": "New Zealand",
+  "Niue": "Niue",
+  "North Korea": "North Korea",
+  "Northern Mariana Islands": "Northern Mariana Islands",
+  "Norway": "Norway",
+  "OCEANIA": "Oceania",
+  "Palestine": "Palestine",
+  "Puerto Rico": "Puerto Rico",
+  "Reunion": "Reunion",
+  "Russia": "Russia",
+  "Saint Helena": "Saint Helena",
+  "Saint Pierre and Miquelon": "Saint Pierre and Miquelon",
+  "Serbia": "Serbia",
+  "Sint Maarten (Dutch part)": "Sint Maarten (Dutch part)",
+  "South America": "South America",
+  "South Korea": "South Korea",
+  "Spain": "Spain",
+  "Syria": "Syria",
+  "Tanzania": "Tanzania",
+  "Tokelau": "Tokelau",
+  "Turks and Caicos Islands": "Turks and Caicos Islands",
+  "Ukraine": "Ukraine",
+  "United Kingdom": "United Kingdom",
+  "United States": "United States",
+  "United States Virgin Islands": "United States Virgin Islands",
+  "Vatican": "Vatican",
+  "Venezuela": "Venezuela",
+  "Vietnam": "Vietnam",
+  "WORLD": "World",
+  "Wallis and Futuna": "Wallis and Futuna",
+  "LATIN AMERICA AND THE CARIBBEAN": "Latin America and the Caribbean",
+  "NORTHERN AMERICA": "Northern America",
```

Comment:
Remember to add the organisational acronym to regions which aren't defined by us, so Northern America (UN) etc. You can be quite harsh in removing regions you don't think will add much value to users, especially if you are adding OWID regions on top of the ones in the dataset.

Reply:
I think keep any that you think are definitely relevant, but I'd try not to have too many flavours of regions. I think OWID continents, WB income groups, and two other organizational regional groupings are a good limit, perhaps three at a push!